### PR TITLE
create harry potter readme file

### PR DESCRIPTION
**Pull request by courtneysprouse**
I wasn't sure if I was supposed to make a contributors.md file? Or if that exists somewhere?

### CODE REVIEW

**Comment by courtneysprouse** [comment]
Can someone review this?

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Courtney Sprouse
```
Comment (aTegart) [inline CONTRIBUTORS.md:1]: Thank you for adding this file!

**Review by aTegart — APPROVED** [review]

**Comment by courtneysprouse** [comment]
@renana Can you review?

**Review by AnnuJoshi — DISMISSED** [review]
Great work, thanks :)

**Review by AnnuJoshi — APPROVED** [review]

**Pull request merged**